frontend/src/InteractBar.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { ReactComponent as ListIcon } from "./svg/list.svg";
+import { ReactComponent as AddPinIcon } from "./svg/file-plus.svg";
+import ListView from "./ListView";
+import LocationForm from "./LocationForm";
+
+function ShowListView() {
+  ReactDOM.render(<ListView />, document.getElementById("root"));
+}
+
+function ShowLocationForm() {
+  ReactDOM.render(<LocationForm />, document.getElementById("root"));
```

Comment:
Nit - I would avoid making calls to ReactDOM.render directly except in index.js. I think the task of changing from one page to another is best done using routing since this isn't a single page application (will address in future task)

Reply:
Sounds good. I'll go ahead and merge it now. Just created task to address this and added to current sprint.